CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
-cmake_minimum_required(VERSION 3.1.3)
+cmake_minimum_required(VERSION 3.3.1)
```

Comment:
Do we need this version bump?

Reply:
It is needed to export the GSL.natvis file, such that it is a dependency on consumers. I don't know what that file is or does, so I don't know if this is the correct way to go.

Reply:
It "enhances" the visual studio debugging experience. See here: https://msdn.microsoft.com/en-us/library/jj620914.aspx

So you can only use the BUILD_INTERFACE generator after 3.3.1?

Reply:
The problem is that the natvis file can't be exposed to consumers through the `MicrosoftGSL::GSL` target because it's not supported before the new version.

Reply:
version 3.3.2 dropped restriction on exporting configuration of targets containing files added with `target_sources(INTERFACE ...)`.
3.3.2 was released in September 2015. It's already quite old.

Reply:
Did you test with an older CMake version to notice the error? It sure has gone unnoticed for a while.

Reply:
@tiagomacarios debugger is used during development of library not in deliveries right? That's why it's used in build not install.